File: Demo/init和initwithframe的区别/init和initwithframe的区别/ViewController.m
//
//  ViewController.m
//  init和initwithframe的区别
//
//  Created by kadis on 2019/8/21.
//  Copyright © 2019 https://github.com/GGTechnology. All rights reserved.
//

#import "ViewController.h"
#import "view.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    view *v = [[view alloc] initWithFrame:CGRectMake(0, 0, 10, 10)];
    
    
    v = [[view alloc] init];
    
    
    
    [self.view addSubview:v];
    
    
}


@end

File: Demo/Extension_UIColor/Extension_UIColor/ViewController.swift
//
//  ViewController.swift
//  Extension_UIColor
//
//  Created by kadis on 2019/9/25.
//  Copyright © 2019 https://github.com/GGTechnology. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet weak var img: UIImageView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        img.backgroundColor = UIColor.init(hexString: "0x123456")
        
    }
    @IBAction func changeColor(_ sender: Any) {
        img.backgroundColor = UIColor.randomColor
    }
}

